src/components/ui/PageContentContainer.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+import { respondFrom, breakpoints } from "../../styles";
```

Comment:
it is recommended to use absolute paths. it's safer that way as it avoids wrong path resolution

Reply:
Ok, i change it